Wikipedia/Legacy Data Migration/WMFLegacyImageDataMigration.swift

```diff
@@ -23,20 +23,8 @@ public class WMFLegacyImageDataMigration : NSObject {
     /// Image controller where data will be migrated.
     let imageController: WMFImageController
 
-    /// List of saved pages which is saved when the tasks are finished processing.
```

Comment:
What about the queue what is the reasoning/effects of removing the background queue? Also there is new logic for re-dipatching to the main queue as well.